hub/util/no_view.py

```diff
@@ -0,0 +1,18 @@
+from hub.util.exceptions import InvalidOperationError
+from typing import Callable
+from functools import wraps
+import hub
+
+
+def no_view(callable: Callable):
+    @wraps(callable)
+    def inner(x, *args, **kwargs):
+        func = callable.__name__
+        if not x.index.is_trivial():
```

Comment:
What if `x` does not have an `index` member?  Or `x.index` doesn't have a `is_trivial` method?

Reply:
this function is to be used internally, only for datasets and tensors